src/hosted-buttons/index.test.js

```diff
@@ -150,5 +155,66 @@ describe("HostedButtons", () => {
     expect(renderMock).toHaveBeenCalledTimes(0);
     expect.assertions(3);
   });
+
+  test("tagline is rendered based on hosted button response", async () => {
+    const renderMock = vi.fn();
+
+    const Buttons = vi.fn(() => ({
+      render: renderMock,
+      isEligible: vi.fn(() => false),
+    }));
+    // $FlowIssue
+    getButtonsComponent.mockImplementationOnce(() => Buttons);
+    const HostedButtons = getHostedButtonsComponent();
+    // $FlowIssue
+    request.mockImplementationOnce(() =>
+      // eslint-disable-next-line compat/compat
+      Promise.resolve({
```

Comment:
can this reuse the previous response from https://github.com/paypal/paypal-checkout-components/blob/71026ef07f05bd150e29a9e0463d5ea5553b2768/src/hosted-buttons/index.test.js#L34 ?

Reply:
Yeah, thought about it, but the tests are changing the value for `tagline` (true, false, undefined). Felt a new object was cleaner than nested spreads and mapping over the array to change one value.